xml/admin_monitoring.xml

```diff
@@ -96,5 +95,435 @@ https://raw.githubusercontent.com/kubernetes/heapster/release-1.3/deploy/kube-co
+    member af7ffa9bb1cb7c67 is healthy: got healthy result from https://caasp-master:2379
+    member cc40a990d09b4705 is healthy: got healthy result from https://caasp-worker-1:2379
+    member fe9b5ee9e1cc3cf7 is healthy: got healthy result from https://caasp-worker-2:2379
+    cluster is healthy
+63008aabc75b471b9a1aa2f64e4d30eb:
+    member af7ffa9bb1cb7c67 is healthy: got healthy result from https://caasp-master:2379
+    member cc40a990d09b4705 is healthy: got healthy result from https://caasp-worker-1:2379
+    member fe9b5ee9e1cc3cf7 is healthy: got healthy result from https://caasp-worker-2:2379
+    cluster is healthy
+   </screen>
+   <para>
+    More information on etcd cluster health can be found in <xref linkend="sec.admin.nodes.graceful_shutdown.etcd"/>.
+   </para>
+   </sect3>
+
+   <sect3 xml:id="sec.admin.monitoring.cluster.health.components">
+    <title>Running Components</title>
+    <para>
+     Check if the cluster has all required components running:
+    </para>
+<screen>&prompt.user;<command>kubectl cluster-info</command>
+
+Kubernetes master is running at https://api.infra.caasp.local:6443
+Dex is running at https://api.infra.caasp.local:6443/api/v1/namespaces/kube-system/services/dex:dex/proxy
+KubeDNS is running at https://api.infra.caasp.local:6443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Tiller is running at https://api.infra.caasp.local:6443/api/v1/namespaces/kube-system/services/tiller:tiller/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+    </screen>
+    <para>
+     You can optionally run <command>kubectl cluster-info dump</command> to
+     obtain a much more detailed output
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.monitoring.health.node">
+   <title>Node Health Checks</title>
+   <para>
+    The basic check if a node is healthy consists of checking if
+    <literal>kubelet</literal> and the CNI (Container Networking Interface) are
+    working properly.
+   </para>
+
+   <sect3 xml:id="sec.admin.monitoring.health.node.kubelet">
+    <title>Check <literal>kubelet</literal> health</title>
+    <para>
+     Is the <literal>kubelet</literal> up and working in this node?
+    </para>
+    <para>
+     The <literal>kubelet</literal> has a port exposed <literal>10250</literal>
+     on all machines; it's possible to perform an HTTP request to the endpoint
+     to find out if the kubelet is healthy on that machine. The
+     expected (healthy) response is a <literal>200 HTTP</literal> and
+     a response body containing <literal>ok</literal>.
+    </para>
+    <para>
+     Endpoint: <literal>https://<replaceable>NODE</replaceable>:10250/healthz</literal> (HTTPS)
+    </para>
+<screen>&prompt.user;<command>curl -i https://localhost:10250/healthz</command>
+ok
+    </screen>
+   </sect3>
+
+   <sect3 xml:id="sec.admin.monitoring.health.node.cni">
+    <title>Check <literal>CNI</literal></title>
```

Comment:
Optionally, this title could use the progressive form to indicate that the user has to / can do something. "Checking CNI"